easybuild/easyblocks/a/acml.py

```diff
@@ -125,7 +127,7 @@ def sanity_check_step(self):
                 inc_files.append(os.path.join(fp, 'include', 'acml%s.h' % inc))
 
             for lib in [suff] + lib_extra:
-                for ext in ['.a', '.so']:
+                for ext in ['.a', shlib_ext]:
```

Comment:
fwiw: while testing my follow-up PR #790, I ran into a bug introduced here;

`shlib_ext` is defined as `so`, not `.so`, so I changed it to use `a` here and `.%s` in the line below, see #790